src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/RunnableProjectGenerator.cs

```diff
@@ -118,58 +118,41 @@ public IParameterSet GetParametersForTemplate(IEngineEnvironmentSettings environ
             localizations = new List<ILocalizationLocator>();
 
-            foreach (IFile file in folder.EnumerateFiles("*" + TemplateConfigFileName, SearchOption.AllDirectories))
+            foreach (IFile file in folder.EnumerateFiles(TemplateConfigFileName, SearchOption.AllDirectories))
             {
-                if (string.Equals(file.Name, TemplateConfigFileName, StringComparison.OrdinalIgnoreCase))
-                {
-                    IFile hostConfigFile = file.MountPoint.EnvironmentSettings.SettingsLoader.FindBestHostTemplateConfigFile(file);
-
-                    if (TryGetTemplateFromConfigInfo(file, out ITemplate template, hostTemplateConfigFile: hostConfigFile))
-                    {
-                        templateList.Add(template);
-                    }
-
-                    continue;
-                }
+                IFile hostConfigFile = file.MountPoint.EnvironmentSettings.SettingsLoader.FindBestHostTemplateConfigFile(file);
 
-                Match localeMatch = localeFileRegex.Match(file.Name);
-                if (localeMatch.Success)
+                if (TryGetTemplateFromConfigInfo(file, out ITemplate template, hostTemplateConfigFile: hostConfigFile))
                 {
-                    string locale = localeMatch.Groups["locale"].Value;
+                    templateList.Add(template);
 
-                    if (TryGetLangPackFromFile(file, out ILocalizationModel locModel))
+                    IDirectory localizeFolder = file.Parent.DirectoryInfo("localize");
+                    if (localizeFolder != null && localizeFolder.Exists)
                     {
-                        ILocalizationLocator locator = new LocalizationLocator()
+                        foreach (IFile locFile in localizeFolder.EnumerateFiles("*" + LocalizationFileSuffix, SearchOption.AllDirectories))
                         {
-                            Locale = locale,
-                            MountPointUri = source.MountPointUri,
-                            ConfigPlace = file.FullPath,
-                            Identity = locModel.Identity,
-                            Author = locModel.Author,
-                            Name = locModel.Name,
-                            Description = locModel.Description,
-                            ParameterSymbols = locModel.ParameterSymbols
-                        };
-                        localizations.Add(locator);
+                            string locale = locFile.Name.Substring(0, locFile.Name.Length - LocalizationFileSuffix.Length);
+
+                            if (TryGetLangPackFromFile(locFile, out ILocalizationModel locModel))
+                            {
+                                ILocalizationLocator locator = new LocalizationLocator()
+                                {
+                                    Locale = locale,
+                                    MountPointUri = source.MountPointUri,
```

Comment:
Do we need MountPointUri and ConfigPlace? I believe they are always same as primary json now.

Reply:
I removed `MountPointUri` as it wasn't used anywhere.
`ConfigPlace` doesn't point to primary json though. And it is used in the code to load localizations.